src/tools/cephfs/top/cephfs-top

```diff
@@ -452,18 +450,21 @@ class FSTop(object):
                         remaining_hlen = 0
                     else:
                         remaining_hlen -= coord[1]
+                    size = 0
                     if key == "READ_IO_SIZES":
-                        global last_read_size
-                        last_size = last_read_size.get(client_id, 0)
-                        size = m[1] - last_size
-                        last_read_size[client_id] = m[1]
+                        if m[1] > 0:
+                            global last_read_size
```

Comment:
there is `last_read_size = {}` on line 100, is this needed here? It would fetch value from that right?

Reply:
It's there to modify the global copy.